server/plugins/renderer/renderer.module.js

```diff
@@ -246,6 +246,10 @@ internals.parseResponse = function (bcAppData, request, response, responseArgs,
                     response: response,
                 }, internals.cacheTTL);
 
+                if (response.headers['set-cookie']) {
```

Comment:
Why do we need this here? We already have it in `getResponse`.

Reply:
bcoz we cache the info but if it is not found in cache we make the call again. There is a bit of duplication which I plan on fixing in follow up PR's. Just wanted to get this out there for now.